.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @ezcater/core-platform @tjwp
+* @ezcater/demand-platform
```

Comment:
There was no service.yml -- do we want one?

Reply:
Since this is a rubygem, let's leave it out!